packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5409,65 +5431,6 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.updateMedia({mediaSettings: this.mediaProperties.mediaDirection});
   }
 
-  /**
-   * Sets the quality level of all meeting media (incoming/outgoing)
-   * @param {String} level {LOW|MEDIUM|HIGH}
-   * @returns {Promise}
-   */
-  setMeetingQuality(level) {
```

Comment:
Can we keep the function and mark it as deprecated and log mentioning to use the local and remote separately

Reply:
/**
   * This is deprecated, please use setLocalVideoQuality for setting local and setRemoteQualityLevel for remote
   * Sets the quality level of all meeting media (incoming/outgoing)
   * @param {String} level {LOW|MEDIUM|HIGH}
   * @returns {Promise}
   * @deprecated After FHD support
   */
   
   We will be bringing back the API and update the js Doc.
   @arun3528, @meta-dipanshu-sharma, @rajeshtezu90

Reply:
@arun3528, incorporated the comments and committed the changes.